internal/cmd/test.go

```diff
@@ -138,3 +139,20 @@ func runCMD(mainFunc func(), args []string) (err error) {
 
 	return err
 }
+
+func RunCommonCMDTests(test *testing.T, mainFunc func(), args []string, expectedExitCode int, index int) (output string, errs error) {
```

Comment:
I'm thinking that we can take this a bit further and also pass expected stdout and stderr (which will usually just be an empty string).